### ISSUE TITLE
Is Full platform 8.6.3 available?

### ISSUE THREAD

**Issue by ghost** [haskell/haskell-platform#318]
Both "Minimal installers" and "Haskell Platform" lead to https://www.haskell.org/platform/windows.html There are only Download Core … buttons

**Comment by gbaz** [comment]
Indeed we’re only producing core builds now. Cf
https://mail.haskell.org/pipermail/haskell-cafe/2018-December/130371.html

This is a good reminder we need to update the page to reflect this.

**Comment by hkdf** [comment]
But https://www.haskell.org/platform/contents.html informs already about content Full platform ?

**Comment by gbaz** [comment]
Indeed. We're maintaining the package listing of recommended versions of the full set. We're just not providing full builds, since their use is now discouraged.

**Comment by yaxu** [comment]
This is a problem for the network package. It doesn't install via the core haskell platform, but does work under the full version (I assume because it ships as part of it). 
https://github.com/haskell/network/issues/361

**Comment by yaxu** [comment]
I take that back, I just tried it out on a windows partition and the network package installed it fine with haskell core. Perhaps the problems others are having is down to https://github.com/haskell/haskell-platform/issues/320 .

**Comment by yaxu** [comment]
I've had a spate of windows people failing to get the haskell platform working with tidalcycles today.
I've just been helping someone using remote admin (teamviewer) and verified the ghc config file is configured correctly.

The error I get from ghci:
```
Prelude> import Sound.Tidal.Context
Prelude Sound.Tidal.Context> tidal <- startTidal superdirtTarget defaultConfig
ghc.exe: | C:\Users\Bryan Green\AppData\Roaming\cabal\x86_64-windows-ghc-8.6.3\network-3.0.1.0-KyWTN8R07Nk13a65qAf8xd\HSnetwork-3.0.1.0-KyWTN8R07Nk13a65qAf8xd.o: unknown symbol `if_nametoindex'
ghc.exe: unable to load package `network-3.0.1.0'
Prelude Sound.Tidal.Context>
```

**Comment by gbaz** [comment]
Yeep. This sounds like it may be a new issue with the 3.0 release of network. Perhaps either @Mistuke or @kazu-yamamoto can weigh in?

(That said it appears this was first added as a function in network 2.7: https://github.com/haskell/network/blob/9f76de6ebbce081eb3c1408061eeb6d560614f54/CHANGELOG.md#version-2700)

One workaround for now might be, if possible, to constrain them to install with a lower version of network?

**Comment by Mistuke** [comment]
This seems to indicate that configure put out something incorrect. That function shouldn't work on windows since `Iphlpapi` isn't declared as a dependency on Windows.

Can you post the `config.log` output from the network build?

**Comment by yaxu** [comment]
Unfortunately I don't have access to the computer any more.

**Comment by Mistuke** [comment]
It seems the problem is this commit https://github.com/haskell/network/commit/3fa7caace378e405ae1dbfafe3f2c0a9ea74c78d#diff-67e997bcfdac55191033d57a16d1408a which removed the configure checks without adding the library dependency for Windows. unfortunately the CI only checks for building network, not if it's usable. I'll fix it and beef up the CI.

**Comment by Mistuke** [comment]
Ah no, it does run the testsuite,we're just missing a test that loads this offending module or uses ghci.

**Comment by yaxu** [comment]
Excellent news, thanks very much! This will make life a lot easier for me.
